base/src/main/java/com/smartdevicelink/proxy/SystemCapabilityManager.java

```diff
@@ -541,6 +555,15 @@ public boolean removeOnSystemCapabilityListener(final SystemCapabilityType syste
 	 * @param subscribe flag to subscribe to updates of the supplied capability type. True means subscribe; false means cancel subscription; null means don't change current subscription status.
 	 */
 	private void retrieveCapability(final SystemCapabilityType systemCapabilityType, final OnSystemCapabilityListener scListener, final Boolean subscribe) {
+		if (currentHMILevel != null && currentHMILevel.equals(HMILevel.HMI_NONE)) {
+			String message = String.format("Attempted to update type: %s in HMI level NONE, which is not allowed. " +
+					"Please wait until you are in HMI BACKGROUND, LIMITED, or FULL before attempting to update any SystemCapabilityType", systemCapabilityType);
+			DebugTool.logError(message);
```

Comment:
Do we need the `message` variable here, or couldn't this simply be inline with the `DebugTool` call?

Reply:
I stored the message in var so I can reuse it when I call `scListener.onError(message)`